test/sql/types/struct/unnamed_struct_comparison.test

```diff
@@ -64,4 +64,35 @@ select 1 from VALUES ((NULL, 1, NULL), (5, 6, 7), (NULL, 2), (4, 5)) t(a, b, c,
 query II
 select a<>b, c<>d from VALUES ((NULL, 1, NULL), (5, 6, 7), (NULL, 2), (4, 5)) t(a, b, c, d) where a<>b and c<>d;
 ----
-true	true
\ No newline at end of file
+true	true
+
+# original issue from author
+
+require inet
```

Comment:
Can we put this in a separate file? That way any tests that get added later will run even if `inet` is not available.